scripts/cnode-helper-scripts/prereqs.sh

```diff
@@ -194,7 +194,7 @@ if [ "$WANT_BUILD_DEPS" = 'Y' ]; then
     $sudo yum -y install curl > /dev/null
     $sudo yum -y update > /dev/null
     echo "  Installing missing prerequisite packages, if any.."
-    pkg_list="python3 coreutils pkgconfig libffi-devel gmp-devel openssl-devel ncurses-libs ncurses-compat-libs systemd systemd-devel libsodium-devel zlib-devel make gcc-c++ tmux git jq gnupg libtool autoconf srm iproute bc tcptraceroute dialog sqlite util-linux xz"
+    pkg_list="python3 coreutils pkgconfig libffi-devel gmp-devel openssl-devel ncurses-libs ncurses-compat-libs systemd systemd-devel libsodium-devel zlib-devel make gcc-c++ tmux git jq gnupg libtool autoconf srm iproute bc tcptraceroute dialog sqlite util-linux xz xxd"
```

Comment:
Could potentially look at something like below instead of xxd:

``` bash
ascwrd=blablabla01234

for ((i=0;i<${#ascwrd};i++));do printf %02X \'${ascwrd:$i:1};done

echo

echo -n ${ascwrd} | xxd -b -ps -c 80
```